### ISSUE TITLE
Legacy config support

### ISSUE THREAD

**Issue by talmo** [talmolab/sleap-nn#63]
Be able to load and parse core SLEAP [`TrainingJobConfig`](https://github.com/talmolab/sleap/blob/c8e3cd09bc6325ff18e2ad352b705f6175880263/sleap/nn/config/training_job.py) JSON files and map them to our new model configs.

**Comment by gqcpm** [comment]
Hi,

For this issue do we want to either:
1. create a sleapnn dependency: reads json --> and have a function output a yaml on sleapnn parameters
or
2. create a sleep dependency: reads json --> convert to trainingjobconfig instance --> save as yaml?

**Comment by talmo** [comment]
We want to read in a legacy JSON file that would normally have created a [`TrainingJobConfig`](https://github.com/talmolab/sleap/blob/c8e3cd09bc6325ff18e2ad352b705f6175880263/sleap/nn/config/training_job.py) object, but instead map the raw JSON-decoded dictionaries on to the equivalent fields in a `OmegaConf` dictionary as per the specifications in our [`docs/config.md`](https://github.com/talmolab/sleap-nn/blob/main/docs/config.md).

Basically it's just mapping dictionary fields onto other dictionary fields, but you don't need to actually save it to a YAML file on disk.

We _definitely_ don't want to depend on SLEAP to create a `TrainingJobConfig` instance.

**Comment by talmo** [comment]
Closing this in favor of #74.
